src/components/group-add-post.js

```diff
@@ -168,14 +174,25 @@ class GroupAddPost extends React.Component {
                   required
                   error={error && errorExist(message, 'text')}
                 >
-                  <TextArea
-                    autoHeight
-                    rows={2}
-                    onChange={this.handleChange}
-                    name='text'
-                    value={this.state.text}
-                    placeholder='Write something here...'
-                  />
+                <Editor
```

Comment:
This is being repeated a lot of times. So make sure you import it from ../fields only.